### PR TITLE
added pair addresses and specific settings for sushiswap and quickswap

### PR DESCRIPTION
**Pull request by SwaroopH**
Noticed it wasn't committed to the repo. Will be hard to track settings changes but I guess we'll move to the decoupled branch soon. Will leave this in for now.

### CODE REVIEW

File: pooler/settings/settings.quickswapv2-example.json [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+        "archival": 180,
+        "connection_init": 2
+    },
+    "module_queues_config":{
+        "pair_total_reserves": {
+            "num_instances": 16
+        }
+    },
+    "epoch": {
+        "height": 10,
```
Comment (xadahiya) [inline pooler/settings/settings.quickswapv2-example.json:88]: Should we change epoch height to 50?

File: pooler/settings/settings.sushiswapv2-example.json [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+    },
+    "module_queues_config":{
+        "pair_total_reserves": {
+            "num_instances": 16
+        }
+    },
+    "epoch": {
+        "height": 10,
+        "head_offset": 2,
+        "block_time": 2
```
Comment (xadahiya) [inline pooler/settings/settings.sushiswapv2-example.json:100]: Block time should be 15 seconds for eth

File: pooler/settings/settings.quickswapv2-example.json [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+    "core_api": {
+        "host": "0.0.0.0",
+        "port": 8002,
+        "auth": {
+            "enabled": false,
+            "header_key": "X-API-KEY"
+        },
+        "public_rate_limit": "10000/day;200/minute;5/second"
+    },
+    "chain_id": 1,
```
Comment (xadahiya) [inline pooler/settings/settings.quickswapv2-example.json:12]: Chain id for polygon is 137

File: pooler/settings/settings.quickswapv2-example.json [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+            {
+                "url": "https://rpc-url",
+                "rate_limit": "500000/day;500/minute;10/second"
+            }
+        ],
+        "archive_nodes": [],
+        "force_archive_blocks": 100,
+        "retry": 1,
+        "request_time_out": 5,
+        "skip_epoch_threshold_blocks": 30
```
Comment (xadahiya) [inline pooler/settings/settings.quickswapv2-example.json:71]: If we change epoch size to 50, this should be 150

**Review by xadahiya — COMMENTED** [review]
Just some minor settings update suggestions, otherwise looks good.

File: pooler/settings/settings.sushiswapv2-example.json (hunk 2 above)
Comment (SwaroopH) [reply pooler/settings/settings.sushiswapv2-example.json:100]: Are these actually being used? I didn't change them as they seems to be the default values in main settings.example.

Comment (xadahiya) [reply pooler/settings/settings.sushiswapv2-example.json:100]: `height` is being used at multiple places and `block_time` is also used in `epoch_broadcast_callback_manager`. Let's update these values.

**Comment by xadahiya** [comment]
Dockerify contains a lot more changes now, closing this. Will create a separate PR with all details.

**Pull request closed without merging**